apps/base/migrations/0003_auto_20160216_2212.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
```

Comment:
I believe migrations are useful for keeping track of schema changes. The migrations here are not really doing anything besides altering fields to have a new default value.